packages/jest-emotion/src/utils.js

```diff
@@ -6,7 +6,7 @@ function getClassNames(selectors, classes) {
 
 function getClassNamesFromTestRenderer(selectors, node) {
   const props = node.props
-  return getClassNames(selectors, props.className || props.class)
+  return props ? getClassNames(selectors, props.className || props.class) : []
```

Comment:
I think it should be `: selectors` instead of `: []`

Reply:
Nice catch, fixed.